cmd/topicctl/subcmd/get.go

```diff
@@ -31,7 +32,24 @@ type getCmdConfig struct {
 
 var getConfig getCmdConfig
 
+type urpCmdConfig struct {
+	topics []string
+}
+
+var urpConfig urpCmdConfig
+
+type opCmdConfig struct {
+	topics []string
+}
+
+var opConfig urpCmdConfig
+
 func init() {
+	log.SetFormatter(&prefixed.TextFormatter{
```

Comment:
This log format and the debug flag should already be getting set by the root command that these subcommands are added to https://github.com/segmentio/topicctl/blob/7ac22690645e4141e6cd134abe68eb67218b8318/cmd/topicctl/subcmd/root.go#L31

Reply:
I think we can remove some duplicate code by adding this to the get subcommands prerun
```go
rootCmd.PersistentPreRun(cmd, args)
```
https://github.com/spf13/cobra/issues/252#issuecomment-571802957